main.py

```diff
@@ -159,30 +163,29 @@ def song_format(
     Returns:
         A string presenting the given song information in a human-readable way.
     """
-    # a valid tag is string with at least one non-whitespace character
-    def is_valid_tag(tag: Optional[str]) -> bool:
-        return tag is not None and tag.strip()
-
     content = ""
-    tags = None
+    artist = ""
+    title = ""
 
     # load tags
     try:
-        tags = TinyTag.get(local_filepath)
-    except TinyTagException:
-        # Ignore and move on
-        pass
+        tags = MutagenFile(local_filepath, easy=True)
+        artist = str(tags.get("artist", [None])[0])
```

Comment:
If "artist" is not a tag then this would return the string `'None'` instead of the type `None`. Is there a reason you wrapped this in `str()`? I think just removing the `str()` would make it fine.

Reply:
I think it was left over from initially testing tag retrieval.